applications/application-registry.md

```diff
@@ -0,0 +1,205 @@
+
+
+### Team Org Repos
+
+- https://github.com/Rewired-one
+- https://github.com/Emanate-official
+- https://github.com/modadao
+- https://github.com/eosphere/heyfio
+
+
+### Team Member Repos
+
+- https://github.com/torbenanderson
+- https://github.com/interwebcoding
+- https://github.com/IanMendozaJaimes
+- https://github.com/tlacloc
+- https://github.com/superstar391
+- https://github.com/djkruitbosch
+
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/torbenanderson1/
+- https://www.linkedin.com/in/interwebcoding/
+- https://www.linkedin.com/in/ian-mendoza-jaimes/
+- https://www.linkedin.com/in/erick-alejandro-casanova-cort%C3%A9s-2a549010a/
+- https://www.linkedin.com/in/collin-wang-1362a51b0/
+- https://www.linkedin.com/in/djkruitbosch/
+
+
+## Development Status
+
+- No development work has started as of this submission
+- EOS Wallet+ Blue Paper
+- Grant proposal reviewed by Greymass team, pre-requisites for further development will be captured under Milstone 1
+- Grant route followed after discussion with and recommendation by Yves
+
+
+## Development Roadmap
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 16 weeks 
+- **Full-Time Equivalent (FTE):**  2.4 FTE
+- **Total Costs:** 229,000 USD
+
+
+### Milestone 1 — Initiate Project & Agree on governance structure and Application Registry metadata
+
+- **Estimated duration:** 3 weeks
+- **FTE:**  0.9
+- **Costs:** 18,081 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | License | MIT |
+| 2. | Project Governance setup | Team setup, governance setup, stakeholder mapping complete. |
+| 3. | Governance agreement | Agreement on how the EOS Mainnet will provide governance on how Application developers can register their applications into the Application Registry. This includes a list of validators who will ensure that apps are following the defined rules. This also defines the rules that apps have to comply with, what metrics they have to support/report and how the app can be updated or deleted. Part of the governance will also consider user input based on their experience with apps. If an app is not behaving as per guidelines, it can be reported. We propose to use a 3/5 multisig with a dedicated governing body. We suggest these are the following entities: greymass, tokenpocket, ENF, ultra and rewired. Part of this deliverable we will also define the application data standards, such as title, description, category, icon, URL, app manifest, etc. |
+
+
+### Milestone 2 — Build Application Registry
+
+- **Estimated Duration:** 15 weeks
```

Comment:
To what degree is this estimation affected by the discovery in Milestone 1 (if at all)?

Does the structure of governance and metadata significantly change this timeline/delivery?

Reply:
Our role in milestone 1 is to bring the 4 wallet providers together and achieve consensus on future governance and metadata requirements. As we will be taking the lead on this, we are not expecting any major impacts on subsequent milestones.